### DISCUSSION TITLE
when iam trying to do expense project mysql setup using ansible i got an error like this.

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1866] [Q&A]
![Uploading Screenshot (38).png…]()

**Comment by Bhashitha19** [comment]
![Screenshot (38)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/1b4ce7f0-d7ec-4412-b623-1830cdd231b9)

**Reply by chaithanya1812** [reply]
Install ansible in the server

**Comment by Bhashitha19** [comment]
![Screenshot (39)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/114d788c-0b95-4d87-aac2-825c4156819f)

**Comment by chaithanya1812** [comment]
## remove  $ in the code
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/75f54060-1b3c-48ba-a30a-36d9d018e252)

## And you are taking some other code reference, so if your code wants to work add this task also.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/f7b0502c-2f6b-460b-88a7-413693b5c2a0)

**Reply by Bhashitha19** [reply]
still it was showing an error like this
![Screenshot (40)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/9b3fb340-b4b6-45f5-8662-45a03dbf0f50)

**Comment by chaithanya1812** [comment]
## issue variable declaration problem, check the variables names [ capital letters & small letters]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/0b64a273-9e3b-41e9-be11-253f3c2de818)

## And also I am unable to login mysql-server can you check the once  security group of mysql-server

**Reply by Bhashitha19** [reply]
mysql ip: 44.202.134.214
![Screenshot (41)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/3b2be7fd-03cd-430d-8401-1d90025814b8)
